### PR TITLE
add desc for tiflash system tables

### PR DESCRIPTION
**Pull request by lidezhu**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: information-schema/information-schema-tiflash-segments.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+title: TIFLASH_SEGMENTS
+summary: 了解 information_schema 表 `TIFLASH_SEGMENTS`。
+---
+
+# TIFLASH_SEGMENTS
+
+`TIFLASH_SEGMENTS` 表提供了 TiFlash 内部数据表的统计信息。
+
+{{< copyable "sql" >}}
+
```
Comment (shichun-0415) [inline information-schema/information-schema-tiflash-segments.md:11]:
```suggestion
```

File: information-schema/information-schema-tiflash-tables.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+title: TIFLASH_TABLES
+summary: 了解 information_schema 表 `TIFLASH_TABLES`。
+---
+
+# TIFLASH_TABLES
+
+`TIFLASH_TABLES` 表提供了 TiFlash 内部数据表的统计信息。
+
+{{< copyable "sql" >}}
+
```
Comment (shichun-0415) [inline information-schema/information-schema-tiflash-tables.md:11]:
```suggestion
```

**Review by flowbehappy — APPROVED** [review]
LGTM

File: information-schema/information-schema-tiflash-tables.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+- `MAX_PACK_COUNT_IN_DELTA`：单个 Delta 层包含 Column File 数量的最大值。
+- `AVG_PACK_COUNT_IN_DELTA`：所有 Delta 层的平均 Column File 数量。
+- `AVG_PACK_ROWS_IN_DELTA`：Delta 层所有 Column File 的平均数据行数。
+- `AVG_PACK_SIZE_IN_DELTA`：Delta 层所有 Column File 的平均数据大小（单位：字节）。
+- `TOTAL_PACK_COUNT_IN_STABLE`：所有 Stable 层的 Pack 总数量。
+- `AVG_PACK_COUNT_IN_STABLE`：所有 Stable 层的平均 Pack 数量。
+- `AVG_PACK_ROWS_IN_STABLE`：Stable 层所有 Pack 的平均数据行数。
+- `AVG_PACK_SIZE_IN_STABLE`：Stable 层所有 Pack 的平均数据大小（单位：字节）。
+- `STORAGE_STABLE_NUM_SNAPSHOTS`：Stable 层 Snapshot 数量。
+- `STORAGE_STABLE_OLDEST_SNAPSHOT_LIFETIME`：Stable 层最老 Snapshot 的持续时间。
```
Comment (shichun-0415) [inline information-schema/information-schema-tiflash-tables.md:122]: 最老 > 最早？

File: information-schema/information-schema-tiflash-tables.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+- `TABLE`：表在 TiFlash 内部的名称。
+- `TIDB_DATABASE`：表所属的数据库的名称。
+- `TIDB_TABLE`：表的名称。
+- `TABLE_ID`：表的内部 ID，该 ID 在一个 TiDB 集群内部唯一。
+- `IS_TOMBSTONE`：表示该表是否处于待回收状态，为 1 表示该表处于待回收状态，为 0 则表示处于正常状态。
+- `SEGMENT_COUNT`：表的 Segment 数量，其中 Segment 为 TiFlash 内部管理数据的单元。
+- `TOTAL_ROWS`：表的总行数。
+- `TOTAL_SIZE`：表的总大小（单位：字节）。
+- `TOTAL_DELETE_RANGES`：表当前 Delete Range 的数量。
+- `DELTA_RATE_ROWS`：表 Delta 层总行数 / 该表的总行数。
```
Comment (shichun-0415) [inline information-schema/information-schema-tiflash-tables.md:91]:
```suggestion
- `DELTA_RATE_ROWS`：表 Delta 层总行数，同时也是该表的总行数。
```

File: information-schema/information-schema-tiflash-segments.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+- `TIDB_DATABASE`：Segment 所属表所属的数据库的名称。
+- `TIDB_TABLE`：Segment 所属表的名称。
+- `TABLE_ID`：Segment 所属表的内部 ID，该 ID 在一个 TiDB 集群内部唯一。
+- `IS_TOMBSTONE`：表示该 Segment 所属表是否处于待回收状态，为 1 表示该表处于待回收状态，为 0 则表示处于正常状态。
+- `SEGMENT_ID`：Segment ID，该 ID 在每张表内部唯一。
+- `RANGE`：Segment 包含数据的范围。
+- `EPOCH`：Segment 的更新版本号，每个 Segment 的版本号单调递增。
+- `ROWS`：Segment 的数据总行数。
+- `SIZE`：Segment 的数据总大小（单位：字节）。
+- `DELTA_RATE`：Delta 层的数据总行数 / Segment 的数据总行数。
```
Comment (shichun-0415) [inline information-schema/information-schema-tiflash-segments.md:71]:
```suggestion
- `DELTA_RATE`：Delta 层的数据总行数，同时也是 Segment 的数据总行数。
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

File: information-schema/information-schema-tiflash-segments.md (hunk 5 above)
Comment (JaySon-Huang) [reply information-schema/information-schema-tiflash-segments.md:71]: 应该是 "- `DELTA_RATE`：Delta 层的数据行数占 Segment 的数据总行数的比例。"

File: information-schema/information-schema-tiflash-segments.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+- `DELTA_MEMTABLE_ROWS`：Delta 层缓存中的数据总行数。
+- `DELTA_MEMTABLE_SIZE`：Delta 层缓存中的数据总大小（单位：字节）。
+- `DELTA_MEMTABLE_COLUMN_FILES`：Delta 层缓存中 Column File 数量。
+- `DELTA_MEMTABLE_DELETE_RANGES`：Delta 层缓存中 Delete Range 数量。
+- `DELTA_PERSISTED_PAGE_ID`：Delta 层数据在磁盘上存储的标识。
```
Comment (JaySon-Huang) [inline information-schema/information-schema-tiflash-segments.md:76]:
```suggestion
- `DELTA_PERSISTED_PAGE_ID`：Delta 层数据在磁盘上存储的 ID。
```

File: information-schema/information-schema-tiflash-segments.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+- `DELTA_PERSISTED_ROWS`：Delta 层已落盘的数据总行数。
+- `DELTA_PERSISTED_SIZE`：Delta 层已落盘的数据总大小（单位：字节）。
+- `DELTA_PERSISTED_COLUMN_FILES`：Delta 层已落盘 Column File 数量。
+- `DELTA_PERSISTED_DELETE_RANGES`：Delta 层已落盘 Delete Range 数量。
+- `DELTA_CACHE_SIZE`：Delta 层缓存大小（单位：字节）。
+- `DELTA_INDEX_SIZE`：Delta 层索引大小（单位：字节）。
+- `STABLE_PAGE_ID`：Stable 层数据在磁盘上存储的标识。
+- `STABLE_ROWS`：Stable 层缓存中的数据总行数。
+- `STABLE_SIZE`：Stable 层缓存中的数据总大小（单位：字节）。
```
Comment (JaySon-Huang) [inline information-schema/information-schema-tiflash-segments.md:85]:
```suggestion
- `STABLE_ROWS`：Stable 层的数据总行数。
- `STABLE_SIZE`：Stable 层的数据总大小（单位：字节）。
```

File: information-schema/information-schema-tiflash-segments.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+- `STABLE_DMFILES`：Stable 层 DMFile 数量。
+- `STABLE_DMFILES_ID_0`：Stable 层第一个 DMFile 在磁盘上存储的标识。
```
Comment (JaySon-Huang) [inline information-schema/information-schema-tiflash-segments.md:87]:
```suggestion
- `STABLE_DMFILES_ID_0`：Stable 层第一个 DMFile 在磁盘上存储的 ID。
```

File: information-schema/information-schema-tiflash-tables.md (hunk 4 above)
Comment (JaySon-Huang) [reply information-schema/information-schema-tiflash-tables.md:91]: "- `DELTA_RATE_ROWS`：表 Delta 层数据总行数占该表总行数的比例。"

File: information-schema/information-schema-tiflash-tables.md [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+- `DELTA_RATE_SEGMENTS`：表所有 Segment 中包含非空 Delta 层的 Segment 的比例。
+- `DELTA_PLACED_RATE`：表 Delta 层数据行中已经完成 Index 构建的比例。
+- `DELTA_CACHE_SIZE`：表 Delta 层缓存大小（单位：字节）。
+- `DELTA_CACHE_RATE`：表 Delta 层数据的缓存比例。
+- `DELTA_CACHE_WASTED_RATE`：表 Delta 层缓存中无效数据的比例。
+- `DELTA_INDEX_SIZE`：Delta Index 的大小（单位：字节）。
```
Comment (JaySon-Huang) [inline information-schema/information-schema-tiflash-tables.md:97]:
```suggestion
- `DELTA_INDEX_SIZE`：Delta Index 占用的内存大小（单位：字节）。
```

File: information-schema/information-schema-tiflash-tables.md (hunk 3 above)
Comment (JaySon-Huang) [reply information-schema/information-schema-tiflash-tables.md:122]: （单位：秒）

File: information-schema/information-schema-tiflash-tables.md [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+- `TABLE`：表在 TiFlash 内部的名称。
+- `TIDB_DATABASE`：表所属的数据库的名称。
+- `TIDB_TABLE`：表的名称。
+- `TABLE_ID`：表的内部 ID，该 ID 在一个 TiDB 集群内部唯一。
+- `IS_TOMBSTONE`：表示该表是否处于待回收状态，为 1 表示该表处于待回收状态，为 0 则表示处于正常状态。
+- `SEGMENT_COUNT`：表的 Segment 数量，其中 Segment 为 TiFlash 内部管理数据的单元。
+- `TOTAL_ROWS`：表的总行数。
+- `TOTAL_SIZE`：表的总大小（单位：字节）。
+- `TOTAL_DELETE_RANGES`：表当前 Delete Range 的数量。
+- `DELTA_RATE_ROWS`：表 Delta 层总行数 / 该表的总行数。
```
Comment (lidezhu) [inline information-schema/information-schema-tiflash-tables.md:89]:
```suggestion
- `DELTA_RATE`：表 Delta 层数据总行数占该表总行数的比例。
```

File: information-schema/information-schema-tiflash-segments.md (hunk 5 above)
Comment (lidezhu) [reply information-schema/information-schema-tiflash-segments.md]: Fixed.

File: information-schema/information-schema-tiflash-tables.md [hunk 11]
```diff
@@ -0,0 +1,132 @@
+---
+title: TIFLASH_TABLES
+summary: 了解 information_schema 表 `TIFLASH_TABLES`。
+---
+
+# TIFLASH_TABLES
+
```
Comment (shichun-0415) [inline information-schema/information-schema-tiflash-tables.md:7]:
```suggestion

> **警告：**
>
> 请不要在生产环境中使用该系统表，因为该表字段信息尚未固定下来，可能会在 TiDB 未来版本中发生变化。

```

File: information-schema/information-schema-tiflash-tables.md [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+- `MAX_PACK_COUNT_IN_DELTA`：单个 Delta 层包含 Column File 数量的最大值。
+- `AVG_PACK_COUNT_IN_DELTA`：所有 Delta 层的平均 Column File 数量。
+- `AVG_PACK_ROWS_IN_DELTA`：Delta 层所有 Column File 的平均数据行数。
+- `AVG_PACK_SIZE_IN_DELTA`：Delta 层所有 Column File 的平均数据大小（单位：字节）。
+- `TOTAL_PACK_COUNT_IN_STABLE`：所有 Stable 层的 Pack 总数量。
+- `AVG_PACK_COUNT_IN_STABLE`：所有 Stable 层的平均 Pack 数量。
+- `AVG_PACK_ROWS_IN_STABLE`：Stable 层所有 Pack 的平均数据行数。
+- `AVG_PACK_SIZE_IN_STABLE`：Stable 层所有 Pack 的平均数据大小（单位：字节）。
+- `STORAGE_STABLE_NUM_SNAPSHOTS`：Stable 层 Snapshot 数量。
+- `STORAGE_STABLE_OLDEST_SNAPSHOT_LIFETIME`：Stable 层最老 Snapshot 的持续时间。
```
Comment (lidezhu) [inline information-schema/information-schema-tiflash-tables.md:120]:
```suggestion
- `STORAGE_STABLE_OLDEST_SNAPSHOT_LIFETIME`：Stable 层最早 Snapshot 的持续时间（单位：秒）。
```

File: information-schema/information-schema-tiflash-tables.md [hunk 13]
```diff
@@ -0,0 +1,132 @@
...
+- `STORAGE_STABLE_OLDEST_SNAPSHOT_THREAD_ID`：Stable 层最老 Snapshot 对应的线程 ID。
+- `STORAGE_STABLE_OLDEST_SNAPSHOT_TRACING_ID`：Stable 层最老 Snapshot 对应的跟踪 ID。
+- `STORAGE_DELTA_NUM_SNAPSHOTS`：Delta 层 Snapshot 数量。
+- `STORAGE_DELTA_OLDEST_SNAPSHOT_LIFETIME`：Delta 层最老 Snapshot 的持续时间。
```
Comment (lidezhu) [inline information-schema/information-schema-tiflash-tables.md:124]:
```suggestion
- `STORAGE_DELTA_OLDEST_SNAPSHOT_LIFETIME`：Delta 层最老 Snapshot 的持续时间（单位：秒）。
```

File: information-schema/information-schema-tiflash-tables.md [hunk 14]
```diff
@@ -0,0 +1,132 @@
...
+- `STORAGE_DELTA_OLDEST_SNAPSHOT_THREAD_ID`：Delta 层最老 Snapshot 对应的线程 ID。
+- `STORAGE_DELTA_OLDEST_SNAPSHOT_TRACING_ID`：Delta 层最老 Snapshot 对应的跟踪 ID。
+- `STORAGE_META_NUM_SNAPSHOTS`：Meta 元信息 Snapshot 数量。
+- `STORAGE_META_OLDEST_SNAPSHOT_LIFETIME`：Meta 元信息最老 Snapshot 的持续时间。
```
Comment (lidezhu) [inline information-schema/information-schema-tiflash-tables.md:128]:
```suggestion
- `STORAGE_META_OLDEST_SNAPSHOT_LIFETIME`：Meta 元信息最老 Snapshot 的持续时间。（单位：秒）
```

File: information-schema/information-schema-tiflash-segments.md [hunk 15]
```diff
@@ -0,0 +1,92 @@
+---
+title: TIFLASH_SEGMENTS
+summary: 了解 information_schema 表 `TIFLASH_SEGMENTS`。
+---
+
+# TIFLASH_SEGMENTS
+
```
[the remark this answers is not in the document]
Comment (shichun-0415) [reply information-schema/information-schema-tiflash-segments.md:7]: @breezewish @lidezhu PTAL

File: information-schema/information-schema-tiflash-tables.md (hunk 3 above)
Comment (lidezhu) [reply information-schema/information-schema-tiflash-tables.md]: Fixed.

**Review by JaySon-Huang — APPROVED** [review]
LGTM

File: information-schema/information-schema-tiflash-tables.md [hunk 16]
```diff
@@ -0,0 +1,136 @@
...
+- `AVG_PACK_COUNT_IN_DELTA`：所有 Delta 层的平均 Column File 数量。
+- `AVG_PACK_ROWS_IN_DELTA`：Delta 层所有 Column File 的平均数据行数。
+- `AVG_PACK_SIZE_IN_DELTA`：Delta 层所有 Column File 的平均数据大小（单位：字节）。
+- `TOTAL_PACK_COUNT_IN_STABLE`：所有 Stable 层的 Pack 总数量。
+- `AVG_PACK_COUNT_IN_STABLE`：所有 Stable 层的平均 Pack 数量。
+- `AVG_PACK_ROWS_IN_STABLE`：Stable 层所有 Pack 的平均数据行数。
+- `AVG_PACK_SIZE_IN_STABLE`：Stable 层所有 Pack 的平均数据大小（单位：字节）。
+- `STORAGE_STABLE_NUM_SNAPSHOTS`：Stable 层 Snapshot 数量。
+- `STORAGE_STABLE_OLDEST_SNAPSHOT_LIFETIME`：Stable 层最早 Snapshot 的持续时间（单位：秒）。
+- `STORAGE_STABLE_OLDEST_SNAPSHOT_THREAD_ID`：Stable 层最老 Snapshot 对应的线程 ID。
```
Comment (lidezhu) [inline information-schema/information-schema-tiflash-tables.md:125]:
```suggestion
- `STORAGE_STABLE_OLDEST_SNAPSHOT_THREAD_ID`：Stable 层最早 Snapshot 对应的线程 ID。
```

File: information-schema/information-schema-tiflash-tables.md [hunk 17]
```diff
@@ -0,0 +1,136 @@
...
+- `STORAGE_STABLE_OLDEST_SNAPSHOT_TRACING_ID`：Stable 层最老 Snapshot 对应的跟踪 ID。
```
Comment (lidezhu) [inline information-schema/information-schema-tiflash-tables.md:126]:
```suggestion
- `STORAGE_STABLE_OLDEST_SNAPSHOT_TRACING_ID`：Stable 层最早 Snapshot 对应的跟踪 ID。
```

File: information-schema/information-schema-tiflash-tables.md [hunk 18]
```diff
@@ -0,0 +1,136 @@
...
+- `STORAGE_DELTA_NUM_SNAPSHOTS`：Delta 层 Snapshot 数量。
+- `STORAGE_DELTA_OLDEST_SNAPSHOT_LIFETIME`：Delta 层最老 Snapshot 的持续时间（单位：秒）。
```
Comment (lidezhu) [inline information-schema/information-schema-tiflash-tables.md:128]:
```suggestion
- `STORAGE_DELTA_OLDEST_SNAPSHOT_LIFETIME`：Delta 层最早 Snapshot 的持续时间（单位：秒）。
```

File: information-schema/information-schema-tiflash-tables.md [hunk 19]
```diff
@@ -0,0 +1,136 @@
...
+- `STORAGE_DELTA_OLDEST_SNAPSHOT_THREAD_ID`：Delta 层最老 Snapshot 对应的线程 ID。
```
Comment (lidezhu) [inline information-schema/information-schema-tiflash-tables.md:129]:
```suggestion
- `STORAGE_DELTA_OLDEST_SNAPSHOT_THREAD_ID`：Delta 层最早 Snapshot 对应的线程 ID。
```

File: information-schema/information-schema-tiflash-tables.md [hunk 20]
```diff
@@ -0,0 +1,136 @@
...
+- `STORAGE_DELTA_OLDEST_SNAPSHOT_TRACING_ID`：Delta 层最老 Snapshot 对应的跟踪 ID。
```
Comment (lidezhu) [inline information-schema/information-schema-tiflash-tables.md:130]:
```suggestion
- `STORAGE_DELTA_OLDEST_SNAPSHOT_TRACING_ID`：Delta 层最早 Snapshot 对应的跟踪 ID。
```

File: information-schema/information-schema-tiflash-tables.md [hunk 21]
```diff
@@ -0,0 +1,136 @@
...
+- `STORAGE_META_NUM_SNAPSHOTS`：Meta 元信息 Snapshot 数量。
+- `STORAGE_META_OLDEST_SNAPSHOT_LIFETIME`：Meta 元信息最老 Snapshot 的持续时间。（单位：秒）
```
Comment (lidezhu) [inline information-schema/information-schema-tiflash-tables.md:132]:
```suggestion
- `STORAGE_META_OLDEST_SNAPSHOT_LIFETIME`：Meta 元信息最早 Snapshot 的持续时间。（单位：秒）
```

File: information-schema/information-schema-tiflash-tables.md [hunk 22]
```diff
@@ -0,0 +1,136 @@
...
+- `STORAGE_META_OLDEST_SNAPSHOT_THREAD_ID`：Meta 元信息最老 Snapshot 对应的线程 ID。
```
Comment (lidezhu) [inline information-schema/information-schema-tiflash-tables.md:133]:
```suggestion
- `STORAGE_META_OLDEST_SNAPSHOT_THREAD_ID`：Meta 元信息最早 Snapshot 对应的线程 ID。
```

File: information-schema/information-schema-tiflash-tables.md [hunk 23]
```diff
@@ -0,0 +1,136 @@
...
+- `STORAGE_META_OLDEST_SNAPSHOT_TRACING_ID`：Meta 元信息最老 Snapshot 对应的跟踪 ID。
```
Comment (lidezhu) [inline information-schema/information-schema-tiflash-tables.md:134]:
```suggestion
- `STORAGE_META_OLDEST_SNAPSHOT_TRACING_ID`：Meta 元信息最早 Snapshot 对应的跟踪 ID。
```

**Pull request merged**